mirrord/intproxy/src/lib.rs

```diff
@@ -141,7 +143,12 @@ impl IntProxy {
         // to requests that have a requirement on the mirrord-protocol version.
         background_tasks.suspend_messages(MainTaskId::LayerInitializer);
         let ping_pong = background_tasks.register_restartable(
-            PingPong::new(Self::PING_INTERVAL, Self::PING_PONG_MAX_RECONNECTS),
+            PingPong::new(
```

Comment:
Can we add a specific error? like ReconnectNotAvailable because operator isn't being used?

Reply:
It's just the ping-pong timeout error essentially, this error would not happen if operator is used (I can add some text about reconnecting in the message)